easy_my_coop_website_taxshelter/controllers/main.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015-2016 Odoo S.A.
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2017-2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
remove non contributors

Reply:
Ok, in this file Jairo doesn't have to be credited.